src/webgpu/shader/execution/expression/call/builtin/atomics/atomicCompareExchangeWeak.spec.ts

```diff
@@ -34,6 +37,264 @@ struct __atomic_compare_exchange_result<T> {
 `
   )
   .params(u =>
-    u.combine('SC', ['storage', 'uniform'] as const).combine('T', ['i32', 'u32'] as const)
+    u
+      .combine('workgroupSize', workgroupSizes)
+      .combine('dispatchSize', dispatchSizes)
+      .combine('mapId', keysOf(kMapId))
+      .combine('scalarKind', ['u32', 'i32'])
```

Comment:
nit: `scalarType`

Reply:
I went ahead and renamed. I think it's alright for this change.